pants.remote.toml

```diff
@@ -2,48 +2,48 @@
 # an account authorized to run the Pants project (you may need to ask a Pants committer for
 # to authorize your account). Then, point to this config file and provide the oauth token like this:
 #
-#  $ ./pants --pants-config-files=pants.remote.ini
+#  $ ./pants --pants-config-files=pants.remote.toml
 #     --remote-oauth-bearer-token-path=<(gcloud auth application-default print-access-token | perl -p -e 'chomp if eof')
 #     --no-v1 --v2 test tests/python/pants_test/util:strutil
 #
 # Remoting does not work for every goal, so you should not permanently point to this ini file, e.g.
 # via an env var; only point to it when you want to remote a specific invocation.
 
-[DEFAULT]
```

Comment:
I changed this to `GLOBAL`. It works to be `DEFAULT` but is not very idiomatic. `DEFAULT` is useful when you want to use string interpolation or apply an option to multiple sections.